### ISSUE TITLE
add support for remote devtools via remotedev

### ISSUE THREAD

**Issue by rob3c** [ngrx/store-devtools#36]
Hi, currently store-devtools will leverage the Redux DevTools Chrome extension if it finds an instance on the `window` object, which is awesome! That does assume that it's running in the same browser with the extension, though.

It'd be nice to also include support for remote debugging so that Cordova/Ionic/etc hybrid app devs can leverage store-devtools during on-device debugging instead of only in the browser where native plugins and such don't work.

For now, I've hacked together a proxy that wraps the [remotedev](https://github.com/zalmoxisus/remotedev) project in [this gist](https://gist.github.com/rob3c/c2c4dcc1116f94901ace179722c5f6d4), which might be useful to see how minimal the addition would need to be to leverage the existing code.

It seems to work well with the Inspector, Log Monitor and Graph, but doesn't work with time-travel and state import due to the open issues with the regular extension, of course. (I don't really know the edge cases my simple wrapper might be missing, though, so any feedback would be welcome.)

Thanks in advance!

**Comment by Gaulomatic** [comment]
Any news on this?

**Comment by danielgek** [comment]
i would like this too, @rob3c thanks for the temp workaround

**Comment by JohannWNel** [comment]
Yes thank you @rob3c very helpfull especially because the extension keeps crashing with large scale projects.
